.github/workflows/tm-load-test.yml

```diff
@@ -0,0 +1,64 @@
+name: Setup chain and apply tm load test
+on:
+  pull_request:
+    branches: [dev]
+
+
+jobs:
+  amd64:
+    name: Setup Chain.
+    runs-on: ubuntu-latest
+    if: github.ref != 'refs/heads/config'
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+        
+      - name: Generate build number
+        id: buildnumber
+        uses: einaregilsson/build-number@v3
+        with:
+          token: ${{secrets.github_token}}
+
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v1
+        with:
+          version: latest
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: Build and push image on dockerhub
+        run: docker buildx build --tag  puneet166/cosmos-node:v-${{ steps.buildnumber.outputs.build_number }} --file  deploy/tm-load-test/Dockerfile --platform  linux/amd64 --push --progress  tty .
+
+      - name: Pull and set a chain.
+        uses: garygrossgarten/github-action-ssh@release
+        with:
+          command: docker run hello-world && docker rm -f $(docker ps -a -q) && docker rmi -f $(docker images -q) && docker run --name cosmos_container -d -p 26656:26656 -p 26657:26657 -p 1317:1317 -p 9090:9090  puneet166/cosmos-node:v-${{ steps.buildnumber.outputs.build_number }} /bin/bash -c "sleep infinity | bash /root/home/master-validator/init.sh"
```

Comment:
What is the sense of "docker run hello-world" here?

Reply:
And why do you need "docker rmi -f" ?

Reply:
This is because if you are building a docker container from an image pulled from dockerhub. It will give us an error if docker container or image with same name already exists
 So before running docker container make sure that container of same name does not exist in that machine. 
We have removed docker hello word.

Reply:
You can use one step to run the image and one step to apply tests, because I don't understand why it is so complicated.